Accepted-RFCs/RFC-013-new-initiative-process.md

```diff
@@ -0,0 +1,124 @@
+# RFC-013: New initiative process
+
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+
+Initially submitted on 24 Aug 2022
+
+## Current status
+
+- [x] Draft
+- [ ] Under discussion
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+The Good Docs Project has an issue of delivery: we haven't delivered a 1.0 of our templates since the inception. And as a volunteer org, we need to keep focus on our mission and goal. It's easy to dilute ourselves if we don't keep ourselves accountable to focus and intentional communal growth.
+
+In other words, we as a project need to make sure we can at least do our core mission well. All new initiatives should 
```

Comment:
```suggestion
In other words, we as a project need to make sure we can at least do our core mission well. All new initiatives should 
```
should what?! Aargh, don't keep me in suspense!

Reply:
I believe it's "All new initiatives should support the organization's core goal/mission."
At least until we're at a point where our mission's execution isn't in its infancy anymore.